storage/rocksdb/CMakeLists.txt

```diff
@@ -82,14 +101,10 @@ ENDIF()
 
 SET(rocksdb_static_libs ${rocksdb_static_libs} ${ZLIB_LIBRARY} "-lrt")
 
-MYSQL_ADD_PLUGIN(rocksdb_se ${ROCKSDB_SOURCES} STORAGE_ENGINE DEFAULT STATIC_ONLY
+MYSQL_ADD_PLUGIN(rocksdb ${ROCKSDB_SOURCES} STORAGE_ENGINE DEFAULT MODULE_ONLY
   LINK_LIBRARIES ${rocksdb_static_libs}
 )
 
-IF(WITH_EMBEDDED_SERVER)
```

Comment:
Yes, I do always build -DWITH_EMBEDDED_SERVER=OFF. Old habit that I never changed since IIRC we didn't ever support embedded server and at one time it would not even compile (maybe some 5.1 days).